virt.te

```diff
@@ -304,6 +309,62 @@ files_type(virt_qemu_ga_data_t)
 type virt_qemu_ga_unconfined_exec_t, virt_file_type;
 application_executable_file(virt_qemu_ga_unconfined_exec_t)
 
+# virtinterfaced
+virt_driver_template(virtinterfaced)
+
+type virtinterfaced_var_run_t;
+files_pid_file(virtinterfaced_var_run_t)
+
+# virtnetworkd
+virt_driver_template(virtnetworkd)
+
+type virtnetworkd_var_run_t;
+files_pid_file(virtnetworkd_var_run_t)
```

Comment:
You can put declaration of *_var_run_t type to the `virt_driver_template()` same as you did with process domains.

Reply:
ok, I will do it